tests/atomistics/job/test_structurecontainer.py

```diff
@@ -0,0 +1,110 @@
+from unittest import TestCase
+from pyiron_atomistics import Project
+from pyiron_contrib.atomistics.atomistics.job.structurecontainer import StructureContainer
+import os.path
+import numpy as np
+
+class TestContainer(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.project = Project(os.path.dirname(__file__))
```

Comment:
If you're setting a project you should probably be inheriting from `TestWithProject` or `TestWithCleanProject` from `pyiron_atomistics._tests`